bin/build_pipeline_heka.sh

```diff
@@ -123,6 +125,10 @@ case $UNAME in
 Darwin)
     # Don't bother trying to build a package on OSX
     make
+    echo "If you see an error like:"
```

Comment:
Yeah but you know it will fail every time on a new build. Also, the same issue should exist on Linux, just LD_LIBRARY_PATH instead.